### PR TITLE
Add checks for test-level links

### PR DESCRIPTION
**Pull request by KyleJu**
Add checks for test-level links and update dummy data after #2172, per https://github.com/web-platform-tests/wpt-metadata/pull/2186

### CODE REVIEW

**Comment by foolip** [comment]
This PR is where the lint mentioned in https://github.com/web-platform-tests/wpt.fyi/pull/2687#discussion_r752948884 should be addressed, probably.

File: css/css-align/gaps/META.yml [hunk 1]
```diff
@@ -7,6 +7,6 @@ links:
         - test: gap-animation-003.html
         - test: gap-animation-004.html
     - url: http://example.org
+      label: example-label1
```
Comment (foolip) [inline css/css-align/gaps/META.yml:10]: If you need at least one label for testing purposes, can I suggest adding a "interop-2021" label to https://github.com/web-platform-tests/wpt.fyi/pull/2686? (And if not, dropping this label seems better.)

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@KyleJu I've added some labels in https://github.com/web-platform-tests/wpt-metadata/pull/2196. If I got it right, the lint changes in this PR should make the tests on that PR pass, right?

File: css/css-align/gaps/META.yml (hunk 1 above)
Comment (KyleJu) [reply css/css-align/gaps/META.yml:10]: I will drop this change

Comment (KyleJu) [reply css/css-align/gaps/META.yml:10]:
```suggestion
```

**Comment by KyleJu** [comment]
> @KyleJu I've added some labels in #2196. If I got it right, the lint changes in this PR should make the tests on that PR pass, right?

not yet. I have to upgrade the deps and add a few checks for  web-platform-tests/wpt.fyi#2687 (comment). I will do it in a followup

**Pull request merged**